packages/editorconfig/src/common/editorconfig-interface.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { KnownProps } from "editorconfig";
+
+export const editorconfigServicePath = '/services/editorconfig';
+
+export const EditorconfigService = Symbol("EditorconfigService");
+export interface EditorconfigService {
+
+    /**
+     * Finds an apropriate editorconfgig properties for a file.
+     */
+    getConfig(file: string): Promise<KnownProps>;
```

Comment:
Please try to follow the convention and use `uri` (as a string) instead of `file` (FS path). Thanks!

Reply:
Will I change only parameter name? Correct?
```
getConfig(uri: string): Promise<KnownProps>;
```

Reply:
Correct, and you have to make sure that a `file://` URI is passed between the frontend and the backend as a string instead of the FS path (but I think that was already the case). Thanks!

Reply:
I used `editor.uri.path.toString()` what gives me absolute path to the file without `file://`. Is it ok to use without prefix `file://`?

Reply:
I would go with the string representation of the URI and on the backend, you can do `FileUri.fsPath(uri)` to get the FS path that you can feed into the `parse` method. So it would be consistent all over the application. Would that be feasible?

Reply:
Done with `uri` and scheme `file:///home/...`